Create passthrough demo with chaos (#60)
conduktor/conduktor-gateway-demos: chaos/Readme.md

```bash
docker compose up --wait --detach
```

We have already created a `username`, called `someUsername`, for our virtual cluster which is provided in the definition of `clientConfig/gateway.properties`, we will use this to interact with Gateway throughout this demo.

> (If you don't want to be useing virtual cluster, please go [here](../passthrough-sasl-plain/Readme.md))

Reviewer: ```suggestion
(If you don't want to use a virtual cluster, please see [here](../passthrough-sasl-plain/Readme.md))
```
